enclave_manager/avalon_enclave_manager/base_enclave_manager.py

```diff
@@ -174,11 +174,11 @@ def validate_request(wo_request):
             False - Otherwise
         """
         try:
-            json.loads(wo_request)
+            json_req = json.loads(wo_request)
         except ValueError as e:
             logger.error("Invalid JSON format found for workorder - %s", e)
-            return False
-        return True
+            return False, None
+        return True, json_req
```

Comment:
Can we convert json string to json object elsewhere or at caller end. Let `validate_request()` only perform validation and return boolean.